hbase/src/main/java/com/yahoo/ycsb/db/HBaseClient.java

```diff
@@ -316,11 +316,12 @@ public int update(String table, String key, HashMap<String,ByteIterator> values)
         Put p = new Put(Bytes.toBytes(key));
         for (Map.Entry<String, ByteIterator> entry : values.entrySet())
         {
+            byte[] value = entry.getValue().toArray();
             if (_debug) {
                 System.out.println("Adding field/value " + entry.getKey() + "/"+
-                  entry.getValue() + " to put request");
+                  Bytes.toString(value) + " to put request");
```

Comment:
What do you think about using Bytes.toStringBinary?
Its Javadoc says "Write a printable representation of a byte array. Non-printable characters are hex escaped in the format \\x%02X, eg: \x00 \x05 etc"
http://hbase.apache.org/apidocs/org/apache/hadoop/hbase/util/Bytes.html#toStringBinary(byte[])